### PR TITLE
Feature/add faqs Document input.nml sections fms_io_nml, interpolator_nml and namsfc

### PR DESCRIPTION
**Pull request by JulieSchramm**
## Description
- Add description of namelist sections fms_io_nml, interpolator_nml and namsfc in input.nml
- Moved documentation of input.nml into a new file InputNML.inc since InputsOutputs.rst was getting too big.

### Issue(s) addressed
Sections of input.nml need to be documented.

## Testing
- Both html and latex builds succeed.
- Non-answer changing.

## Dependencies
None.

### CODE REVIEW

**Comment by climbfuji** [comment]
@ligiabernardet if you can review and approve this PR, I will make sure that it gets merged into the ufs-weather-model public release branch. Thanks!

File: doc/UsersGuide/source/FAQ.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+       qsub job_card
+
+============================================
+How do I change the length of the model run?
+============================================
+In your run directory, there is a file named ``model_configure``.  Change the
+variable ``nhours_fcst`` to the desired number of hours.
+
+=====================================================
+How do I change the model output to netcdf or nemsio?
```
Comment (ligiabernardet) [inline doc/UsersGuide/source/FAQ.rst]:
```suggestion
How do I change the model output to netCDF or NEMSIO?
```

Comment (ligiabernardet) [reply doc/UsersGuide/source/FAQ.rst]: Is this only valid if the write component is being used?

**Review by ligiabernardet — CHANGES_REQUESTED** [review]
Thank you @JulieSchramm, this added documentation is very helpful.

**Review by ligiabernardet — APPROVED** [review]

**Pull request merged**